### DISCUSSION TITLE
Where do I set CUDA PyTorch environment variables?

### DISCUSSION THREAD

**Discussion by abdullahcfix** [vladmandic/sdnext#2019] [Q&A]
In A1111, people add their launch arguments and set environment variables in the webui-user.bat, but as a user of SD.Next since May, I haven't figured out how to set the environment variables yet. I'm getting an OOM error but only by 40MB, so I'm trying to see if I can tinker with the option that the error message mentions, `max_split_size_mb`.

Where can I modify all this stuff?

Thanks!

**Comment by brknsoul** [comment]
https://github.com/vladmandic/automatic/issues/2013#issuecomment-1679622049 should help.
